students/kaczmarek_katarzyna/lesson_02_flow_control/lost_card.py

```diff
@@ -0,0 +1,9 @@
+n = int(input())
+sum_all_cards = 0
+
+for i in range(1, n + 1):
```

Comment:
[opcjonalnie] to mozna przyśpieszyc i zamiast liczyć w pętli, skorzystać ze wzoru na sumę szeregu.